.azure-pipelines/client.yml

```diff
@@ -39,7 +39,18 @@ jobs:
           version: 2.1.503
 
       - task: DotNetCoreCLI@2
-        displayName: 'Build & Test'
+        displayName: 'Build & Test (no live tests)'
+        condition: eq(variables['System.TeamProject'], 'public')
+        inputs:
+          command: test
+          projects: '$(ProjectFile)'
+          arguments: --filter TestCategory!=Live
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Build & Test (with live tests)'
+        condition: ne(variables['System.TeamProject'], 'public')
+        env:
+          AZ_CONFIG_CONNECTION: $(AzConfigConnectionString)
```

Comment:
The pipeline needs to know where to get the `AzConfigConnectionString` value from. We will need to add something like: https://github.com/Azure/azure-sdk-for-net-lab/blob/master/.azure-pipelines/azure-pipelines.yml#L1
and add the corresponding key from KV tot he internal pipeline definition